draft-ietf-ccwg-rfc5033bis.md

```diff
@@ -194,6 +194,12 @@ of their proposals for publication in the RFC series to allow others
 to concretely understand and investigate the wealth of proposals in
 this space.
 
+This document is meant to reduce the barriers to entry for new congestion
+control work. As such, proponents should not interpret these criteria as a
+checklist of requirements before approaching the IETF. Instead, proponents
+are encouraged to have thought about most of these issues beforehand, and have
```

Comment:
Weird use of tenses. Might be simpler to just say, "Instead, proponents
are encouraged to think about most of these issues beforehand"